core/src/main/scala/org/ensime/core/Analyzer.scala

```diff
@@ -163,12 +189,21 @@ class Analyzer(
     case RestartScalaCompilerReq(id, strategy) =>
       restartCompiler(strategy, id)
 
+    case SuspendAnalyzer =>
+      val now = clock.millis()
```

Comment:
I don't know what the time tracking is all about... just ask to shutdown and then become suspended.